### ISSUE TITLE
NTR: ? "plasma membrane lipid organization" 

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#28900]
Please provide as much information as you can: 

* **Suggested term label:**

I want to annotate the process of duc1

In this study, we identified the previously uncharacterized S. pombe protein Duc1 based on its proximity to the PM-localized PI-5 kinase Its3 and the Its3 binding partner Opy1. We determined that Duc1 promotes the proper PM localization of Its3 and therefore functions to maintain proper lateral PM lipid composition, CR anchoring, and medial septation.

I think the primary role here is local PM lipid composition, and so 
"plasma membrane organization" seems appropriate. However I want it to be specific for membrane lipids organization (because this term has children including  " plasma membrane proton-transporting ATP synthase complex assembly" " integrin biosynthetic process" and the organization of specific structures "T-tubule organization" "cornified envelope assembly" or " plasma membrane fusion"

@pgaudet  do you think  new   a term like 
"plasma membrane lipid organization"  or ""plasma membrane lipid organization" would be OK?
The only similar term is "GO:0097036  (https://www.ebi.ac.uk/QuickGO/services/ontology/go/terms/GO:0097036/complete)
regulation of plasma membrane sterol distribution"

but I don't want a regulation term. This isn't regulating in the GO sense, it's "affecting" but it does seem to be part of the process

* **Definition (free text)**

* **Reference, in format PMID:#######**
(REQUIRED)

* **Gene product name and ID to be annotated to this term**

* **Parent term(s)**

* **Children terms (if applicable)** Should any existing terms that should be moved underneath this new proposed term?

* **Synonyms (please specify, EXACT, BROAD, NARROW or RELATED)**

* **Cross-references**
* For enzymes, please provide RHEA and/or EC numbers. 
* Can also provide MetaCyc, KEGG, Wikipedia, and other links.

* **Any other information**

**Comment by pgaudet** [comment]
Hi @ValWood 

Very interesting question! I wonder if there is a *coordinated* process for 'regulation of sterol distribution'  (2 EXP, pombase and SGD))- I would propose to obsolete this and only have PM organization, I dont think individual components are organized via specific pathways; these represents specific MFs (various flippases, for example), 

Then we have 3 types of children: 
1. the molecule being 'organized' (=[plasma membrane phospholipid scrambling](http://purl.obolibrary.org/obo/GO_0017121) and [regulation of plasma membrane sterol distribution](http://purl.obolibrary.org/obo/GO_0097036))
2. PMs of specific cells ([cornified envelope assembly](http://purl.obolibrary.org/obo/GO_1903575) and [myelin maintenance](http://purl.obolibrary.org/obo/GO_0043217), [sperm plasma membrane disassembly](http://purl.obolibrary.org/obo/GO_0035045))
3. processes that have_part some PM reorganization ([plasma membrane fusion](http://purl.obolibrary.org/obo/GO_0045026), [plasma membrane repair](http://purl.obolibrary.org/obo/GO_0001778), [plasma membrane tubulation](http://purl.obolibrary.org/obo/GO_0097320)) 


I think only #2 should be valid, on the condition that these are not just PM organization that occur in a specific cell, but that these would be different pathways. I haven't looked at the annotations so I don't know. 
----

Specifically about [regulation of plasma membrane sterol distribution](http://purl.obolibrary.org/obo/GO_0097036): 

One of the papers cited as a def xref is (https://pubmed.ncbi.nlm.nih.gov/20823909/); it seems to talk about other lipids, in addition to sterols. The other paper is https://pubmed.ncbi.nlm.nih.gov/18441123/, which seems to describe a phenotype. 

These papers do not convincingly show the need for that term.


----

Proposed action: 

The 'easiest' would be to obsolete 'regulation of sterol distribution' and annotate your gene to PM organization. 

(but really the while branch of GO:0097035 regulation of membrane lipid distribution looks like it should be merged up to 'membrane organization') 
Depends if you want ta project or a quick fix !

**Comment by ValWood** [comment]
I agree with you clean up suggestions, especially for the 'regulation' terms.

but I also think there is some value in the organization of specific lipids because the lipid at a specific location determine the processes. For if you alter the PtdIns(4,5)P2 composition you alter the positioning to the positioning of the cytokinetic ring.

There may be other ways to model this. I will have a go once I have some clarification from the author.
It will be after my break

**Comment by ValWood** [comment]
I'm going to look into this some more when I get back. I think it is "causally upstream" of cytokinesis and that is how I am planning to represent it in the GO cam (it provides the right "environment" for ring formation. I will make a GO CAM for this when I get back and discuss  further with the authors.

**Comment by ValWood** [comment]
Closing I don't require this term
